### PR TITLE
added data for Nersingen

### PR DESCRIPTION
**Pull request by looniverse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ A small visualization of the weekly markets in different cities. Available at: h
 |[München][muenchen-wikipedia]|[City of München][muenchen-markets]|
 |[Münster][muenster-wikipedia]|[City of Münster][muenster-markets]|
 |[Neu-Ulm][neu-ulm-wikipedia]|[City of Neu-Ulm][neu-ulm-markets]|
+|[Nersingen][nersingen-wikipedia]|[City of Nersingen][nersingen-markets]|
```
Comment (johnjohndoe) [inline README.md]: Please keep the alphabetic order of the list. Same at the links section at the bottom of the file. Feel free to overwrite this commit via `git commit --amend` and `git push --force origin market-data-nersingen`.

File: cities/nersingen.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                "title": "Nersinger Wochenmarkt",
+                "location": "Rathausplatz Nersingen",
+                "opening_hours": "Fr 13:00-17:00"
+            },
+            "type": "Feature"
+        }
+    ],
+    "metadata": {
+        "data_source": {
+            "title": "Daten von der Gemeinde Nersingen",
```
Comment (johnjohndoe) [inline cities/nersingen.json]: Please use "Gemeinde Nersigen" only.

**Comment by johnjohndoe** [comment]
Cool. Can you please merge the second commit into the first commit? Let me know if you need help with this. You need to use the Git command line client to do this.

**Comment by looniverse** [comment]
Hi Tobias,
I need your help please. 
Thank you for your patience. 

Regards Thomas

**Comment by k-nut** [comment]
@johnjohndoe we could also just use githubs [merge squashing](https://help.github.com/articles/about-pull-request-merge-squashing/) feature here, couldn't we?

**Comment by johnjohndoe** [comment]
@k-nut Good point. However, since `master` has grown further meanwhile - this branch has to be rebased. @looniverse I sent you an invite to our Slack community via email. Did you receive it?

**Comment by johnjohndoe** [comment]
@looniverse Welcome to the map! :rocket: Please promote the website in your city, see #38.

**Pull request merged**